src/plays/zoomlogin/components/Zoom/ZoomPage.js

```diff
@@ -0,0 +1,180 @@
+import zoomImg from '../../assets/zoom-img.png';
+import zoomAdImg from '../../assets/zoomAd.png';
+
+function ZoomPage() {
+  const [isHelp, setIsHelp] = useState(false);
+  const [isHovered, setIsHovered] = useState(false);
+  const [isChatbotClose, setIsChatbotClose] = useState(true);
+  const [isChatbotHeaderClose, setIsChatbotHeaderClose] = useState(true);
+
+  const handleHover = () => {
+    setIsHovered(!isHovered);
+  };
+
+  const handleChatbotModal = () => {
+    setIsChatbotClose(!isChatbotClose);
+    setIsChatbotHeaderClose(false);
+    setIsHelp(false);
+  };
+
+  const toggleHeaderActionMenuModal = () => {
+    setIsChatbotHeaderClose(!isChatbotHeaderClose);
+  };
+
+  const validateInput = (message) => {
+    if (!message.trim()) {
+      return false;
+    }
+
+    return true;
+  };
+
+  useEffect(() => {
+    const inputElement = document.querySelector('.react-chatbot-kit-chat-input');
+
+    if (inputElement) {
+      inputElement.placeholder = 'Write a message (e.g. reset password, easter egg dog 🐶 )';
+      inputElement.blur();
+    }
+
+    const formElement = document.querySelector('.react-chatbot-kit-chat-input-form');
+
+    if (formElement) {
+      const iconsContainer = document.createElement('div');
+      iconsContainer.style.display = 'flex';
+      iconsContainer.style.alignItems = 'center';
+
+      const fileButton = document.createElement('button');
+      fileButton.style.background = 'none';
+      fileButton.style.border = 'none';
+      fileButton.style.cursor = 'not-allowed';
+      fileButton.disabled = true;
+
+      const fileIconElement = document.createElement('img');
+      fileIconElement.src = `${fileUpload}`;
+      fileIconElement.alt = 'file upload';
+      fileIconElement.style.marginLeft = '14px';
+      fileIconElement.style.marginTop = '14px';
+      fileIconElement.style.width = '22px';
+      fileIconElement.style.padding = '3px';
+
+      fileIconElement.addEventListener('mouseover', () => {
+        fileIconElement.style.backgroundColor = '#52528017';
+        fileIconElement.style.borderRadius = '2px';
+      });
+
+      fileIconElement.addEventListener('mouseout', () => {
+        fileIconElement.style.backgroundColor = '';
+        fileIconElement.style.border = 'none';
+      });
+
+      fileButton.appendChild(fileIconElement);
+      iconsContainer.appendChild(fileButton);
+
+      const emojiButton = document.createElement('button');
+      emojiButton.style.background = 'none';
+      emojiButton.style.border = 'none';
+      emojiButton.style.cursor = 'not-allowed';
+      emojiButton.style.width = '22px';
+      emojiButton.disabled = true;
+
+      const emojiIconElement = document.createElement('img');
+      emojiIconElement.src = `${emojiIcon}`;
+      emojiIconElement.alt = 'emoji icon';
+      emojiIconElement.style.marginLeft = '2px';
+      emojiIconElement.style.marginTop = '14px';
+      emojiIconElement.style.padding = '3px';
```

Comment:
please avoid direct DOM manipulation

Reply:
I deleted the direct DOM manipulation codes.

Reply:
ok